$NetBSD: patch-lib_build_OCaml.om,v 1.2 2018/02/12 11:03:35 dholland Exp $

Fix build failure. Apparently $(c_comp) is an array and it needs
to be a string.

--- lib/build/OCaml.om~	2017-10-29 11:36:41.000000000 +0000
+++ lib/build/OCaml.om
@@ -181,7 +181,7 @@ private.get_bytecomp_c_comp() =
     # Figure out the params for the C compiler
     #
     private.c_comp = $(get_c_comp)
-    if $(equal X$(c_comp)X, XX)
+    if $(equal X$(string $(c_comp))X, XX)
         private.bytecomp_c_comp = $(get_bytecomp_c_comp)
         OCAML_CC = $(nth-hd 1, $(bytecomp_c_comp))
         OCAML_CFLAGS = $(nth-tl 1, $(bytecomp_c_comp))
